src/hooks/useOrders.ts

```diff
@@ -64,7 +64,8 @@ export function useOrders(): Result {
     const fetchOrders = async (offset: number): Promise<void> => {
       // isLoading is the important one
       // controls ongoing fetching chain
-      if (!userAddress || !networkId || !isLoading) {
+      if (!userAddress || !networkId /*|| !isLoading*/) {
+        // TODO: Do not merge with this, just for doing a quick test!!
```

Comment:
?

Reply:
Yes! :P 

This is so this don't get merged until that issue is fixed.
I commented that so Michel can see the orders. The problem is that there was a race condicion loading the wallet, and the fix broke the mocks for the orders. 

This is a temporary fix. You can still review the other parts of this PR.

Reply:
@anxolin we still have that. Guess this should be reverted?